tests/expectations/tests/class_nested.rs

```diff
@@ -77,7 +77,7 @@ fn bindgen_test_layout_A() {
 #[repr(C)]
 #[derive(Debug, Default, Copy, Clone, Hash, PartialEq, Eq)]
 pub struct A_C {
-    pub baz: ::std::os::raw::c_int,
+    baz: ::std::os::raw::c_int,
```

Comment:
This will break many consumers, I think we should put it behind an opt-in flag, wdyt?

Reply:
SGTM; done.